source/_integrations/enasolar.markdown

```diff
@@ -0,0 +1,95 @@
+---
+title: EnaSolar Solar Inverter
+description: Instructions on how to connect your EnaSolar GT Solar Inverter to Home Assistant.
+ha_category:
+  - Energy
+ha_iot_class: Local Polling
+ha_release: '0.1.0'
+ha_codeowners:
+  - '@geustace'
+ha_domain: enasolar
+ha_platforms:
+  - sensor
+---
+
+The `enasolar` sensor will poll a EnaTel EnaSolar GT solar inverter.  EnaTel are no longer in the solar market but their inverters were popular in Australia and New Zealand.
+
+This sensor uses the inverter's web interface. To use it, you have to be able to connect to the solar inverter from your favorite web browser. Try http://your-inverter/settings.html
+
+Note: Polling too frequently caused the web server in the inverter to become unresponsive, to reset turn the AC switch on the inverter Off and On again. The current implementation polls the meters every minute and the daily data every 10 minutes. Polling is also reduced at night.
+
+
+## Configuration
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: enasolar
+    host: IP_ADDRESS_OF_DEVICE
+```
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configuration.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>